coral/src/app/features/topics/acl-request/fields/RemarksField.tsx

```diff
@@ -0,0 +1,11 @@
+import { Textarea } from "src/app/components/Form";
+
+const RemarksField = () => (
```

Comment:
These extremely simple components seem like a superfluous abstraction, but as this field is used in both `TopicProducerForm` and `TopicConsumerForm`, I preferred having it separate so that we don't need changes in two spots when editing it.